Multiple Displays (#64)
TomWhitwell/SlowMovie: README.md

config file values, which in turn override defaults.
```

### E-ink Display Drivers

The guide for this program uses the [7.5-inch Waveshare display](https://www.waveshare.com/product/displays/e-paper/epaper-1/7.5inch-e-paper-hat.htm), this is the device driver loaded by default. It is possible to specify other devices using the command line `-e` option. Currently most Waveshare displays are compatible but we hope to have others in the near future. An example of running Slowmovie with a 4.2inch Waveshare display instead would be:

```
python3 slowmovie.py -e waveshare_epd.epd4in2b_V2

```

> You can view a list of compatible e-ink devices by [clicking here](https://github.com/robweber/vsmp-epd/blob/main/README.md#displays-implemented).

Reviewer: ```suggestion
You can view a list of compatible e-ink devices on the [VSPM-EPD repo](https://github.com/robweber/vsmp-epd/blob/main/README.md#displays-implemented).
```

More context in link text (https://accessibility.psu.edu/linktext/)
